docs/CONTRIBUTING.md

```diff
@@ -14,6 +14,21 @@ Requirements to develop SkyEye:
 
 ## Build
 
+### General
+
+If you do not use Windows as a platform but want to use VSCode for Go development you need to adjust the workspace settings of Vscode, otherwise the go extensions won't work:
```

Comment:
Looks like this can't be automated due to https://github.com/microsoft/vscode/issues/5595, however I'll at least add an alternate file for macOS/Linux that can be swapped in.

Reply:
Done.